services/web/server/src/simcore_service_webserver/licenses/_common/models.py

```diff
@@ -15,14 +21,44 @@
 from models_library.rest_pagination import PageQueryParameters
 from models_library.users import UserID
 from models_library.wallets import WalletID
-from pydantic import BaseModel, ConfigDict, Field
+from pydantic import BaseModel, ConfigDict, Field, PositiveInt
 from servicelib.request_keys import RQT_USERID_KEY
 
 from ..._constants import RQ_PRODUCT_KEY
 
 _logger = logging.getLogger(__name__)
 
 
+class LicensedItem(BaseModel):
+    licensed_item_id: LicensedItemID
+    licensed_resource_name: str
+    licensed_resource_type: LicensedResourceType
+    licensed_resource_data: dict[str, Any]
+    pricing_plan_id: PricingPlanId
+    created_at: datetime
+    modified_at: datetime
+    model_config = ConfigDict(
```

Comment:
I would move domain models to `models_library.licenses` module. `licenses._common.models` is for domain models that have special needs e.g. extra dependencies, due to validators, etc